### PR TITLE
Route messages in agent class

### PR DESCRIPTION
**Pull request by dbluhm**
Also moved message types to relevant module definitions rather than in a central message_types.py.

### CODE REVIEW

File: python/indy-agent.py [hunk 1]
```diff
@@ -51,27 +49,23 @@
...
-    'admin_walletconnection': AdminWalletConnection(WEBAPP['agent']),
-    'basicmessage': BasicMessage(WEBAPP['agent']),
-    'admin_basicmessage': AdminBasicMessage(WEBAPP['agent'])
-}
-WEBAPP['agent'].modules = WEBAPP['modules']
+AGENT = Agent()
+
+WEBAPP['agent'] = AGENT
+
+AGENT.register_module(Admin(AGENT))
```
Comment (TelegramSam) [inline python/indy-agent.py]: How do you feel about having the register_module() method instantiate the module internally, passing itself (the agent) as the arg? Would look like this on the outside:
`AGENT.register_module(Admin)`

File: python/agent.py [hunk 2]
```diff
@@ -24,6 +25,14 @@ def __init__(self):
         self.ui_socket = None
         self.initialized = False
         self.modules = []
+        self.family_router = FamilyRouter()
+
+    def register_module(self, module):
+        self.modules.append(module)
+        self.family_router.register(module.FAMILY, module)
```
Comment (TelegramSam) [inline python/agent.py]: I think we've made a past mistake here, and we are routing messages by just the family name, and not version or base DID. We should probably use module.BASE.

File: python/modules/admin.py [hunk 3]
```diff
@@ -6,15 +6,20 @@
 from router.simple_router import SimpleRouter
 from agent import Agent
 from message import Message
-from message_types import ADMIN
 from . import Module
 
 class Admin(Module):
+    FAMILY = "admin"
+    VERSION = "1.0"
+    BASE = "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/" + FAMILY + "/" + VERSION + "/"
```
Comment (TelegramSam) [inline python/modules/admin.py]: should BASE be FAMILY, and FAMILY be FAMILY_NAME?

File: python/agent.py (hunk 2 above)
Comment (dbluhm) [reply python/agent.py]: That's a good point. I think I'll have to tweak the family router a little to support that.

**Pull request merged**